### ISSUE TITLE
Allow remote caldav/carddav servers

### ISSUE THREAD

**Issue by tuxis-ie** [nextcloud/server#8469]
Nextcloud is great for sharing files, and supports Caldav & Carddav services in the process! This is great, but there is one feature missing, email. Other products are great for handling email, and support Caldav & Carddav in the proces! This is also great, but the feature of handling files is missing.

A possible solution would be for Nextcloud to support external Caldav calendars and Carddav addressbooks. Read/write would be enough. Sharing features would need to be handled by the originating product.

It would be so cool to make Nextcloud a one-stop-shop for users.

**Comment by MorrisJobke** [comment]
Is this about the calendar app to connect to a different caldav server or about adding a remote calendar into the Nextcloud CalDAV?

cc @georgehrke

**Comment by georgehrke** [comment]
if @tuxis-ie is the one I talked to in IRC a couple of days ago its about adding CalDAV calendars from different sources to Nextcloud. Basically what external storage does for files.

And it should be implemented in the dav app, so not only available for the calendar app but for all CalDAV applications that connect to the Nextcloud server.

**Comment by georgehrke** [comment]
Generally speaking: This would be a nice addition, but it's really really low priority. Not anytime soon.

**Comment by georgehrke** [comment]
This is actually a duplicate of #252

**Comment by danielsteiner** [comment]
Any updates on this? 
With all-in-one mailsolutions that come with card / caldav servers it's quite useless to have multiple card/caldav servers running. The big advantage with Nextcloud is, that it has Mail, Contacts, Calendars, Files and much more in one interface and I really don't want to have duplicate data in my Groupware and Nextcloud and I don't want multiple apps for files and contacts / calendars / mail either (just as many others)

**Comment by georgehrke** [comment]
@danielsteiner https://github.com/nextcloud/server/issues/252#issuecomment-531479395

tl;dr: Sure, would be nice to have, but not on any roadmap anytime soon. As always: Contributions welcome!

**Comment by danielsteiner** [comment]
Will take a look at this then.

**Comment by nikinname** [comment]
For me also would be important. In this way nextcloud could be a single interface for mail (and contacts) and all other functions.


Thank

**Comment by kiv57** [comment]
+1
I need this feature more than ever as TB becomes less and less usable with each update...
